graphql_compiler/compiler/expressions.py

```diff
@@ -976,6 +978,8 @@ def to_gremlin(self):
             u'not_contains': (u'contains', negated_dotted_operator_format),
             u'intersects': (u'intersect', intersects_operator_format),
             u'has_substring': (u'contains', dotted_operator_format),
+            u'starts_with': (u'==~', immediate_operator_format),
```

Comment:
I think going for regex here is overkill. Since OrientDB Gremlin is just a Groovy interpreter with extra functions, let's just use the existing `startsWith/endsWith` functions on strings if possible?
https://blogs.oracle.com/fadevrel/useful-groovy-string-functions

This will also help us avoid adding to the set of `legal_strings_with_special_chars` in the file below, which is a nice bonus.

Reply:
cool! I didn't know Gremlin had these functions. I will switch to that implementation.